data/card_catalogue/npc/rebels.json

```diff
@@ -4,40 +4,51 @@
     "upgrade": "Empty the Clip",
     "tier": "blue",
     "details": {
-      "rules": "You can use an additional ranged attack",
+      "rules": {
+        "active": "You can use an additional ranged attack.",
+        "passive": ""
+      },
       "flavour": "Rebel assault rifles are equipped with manual and fully automatic firing modes.",
-      "image": "assets/core/card/illustration/Reload.png"
+      "image": "assets/core/card/illustration/Quick Shot.png"
     }
   },
   {
     "title": "Empty the Clip",
-    "upgrade": "Surpression Maneuver",
+    "upgrade": "Suppression Maneuver",
     "tier": "green",
     "details": {
-      "rules": "You can use two additional ranged attacks. Return a Card with Blue Rarity to your hand.",
-      "flavour": "Rebel soliders employ distraction devices to buy time to reload.",
-      "image": "assets/core/card/illustration/Reload.png"
+      "rules": {
+        "active": "You can use two additional ranged attacks. Return a Card with Blue Rarity to your hand.",
+        "passive": ""
+      },
+      "flavour": "Rebel soldiers employ distraction devices to buy time to reload.",
+      "image": "assets/core/card/illustration/Empty the Clip.png"
     }
   },
   {
-    "title": "Surpression Maneuver",
+    "title": "Suppression Maneuver",
     "upgrade": "Bullet Rain",
     "tier": "red",
     "details": {
-      "rules": "You can use four additional ranged attacks. Return a Card with Green Rarity to your hand.",
+      "rules": {
+        "active": "You can use four additional ranged attacks. Return a Card with Green Rarity to your hand.",
```

Comment:
For cases like this where we're talking about other cards of a specific tier on a card I'd prefer to have it be

> return <del>a Card with Green Rarity</del> <ins>a green</ins> to your hand

Any thoughts @DuncanUszkay1? Wouldn't feel comfortable making that change to the catalogue without your approval

Reply:
Makes sense to me 👍

Reply:
tracked in

- #122